tests/test_ssim.py

```diff
@@ -2,96 +2,81 @@
 import itertools
 import pytest
 import tensorflow as tf
-
+import numpy as np
+from PIL import Image
 from piq import SSIMLoss, MultiScaleSSIMLoss, ssim, multi_scale_ssim
+from typing import Tuple, List, Any
 
 
 @pytest.fixture(scope='module')
 def prediction() -> torch.Tensor:
-    return torch.rand(3, 3, 256, 256)
+    return torch.rand(3, 3, 161, 161)
```

Comment:
Is this a minimal size for MS-SSIM? 
If so, may be add it somewhere in the documentation?

Reply:
Yes, we have limitations due to downscaling inside the MS-SSIM. Added to doc.